.github/workflows/generate-base-image.yml

```diff
@@ -0,0 +1,206 @@
+name: Generate and push jvm-runtime base image to Dockerhub
+
+on:
+  workflow_dispatch:
+    inputs:
+      sem-ver:
+        type: choice
+        description: Version bump
+        required: false
+        default: None
+        options:
+          - Existing
```

Comment:
Use the same pattern here as well.  First letter should be simple case

Reply:
Done